scripts/run_e2e_tests.py

```diff
@@ -34,7 +34,7 @@
 from scripts import setup
 from scripts import setup_gae
 
-CHROME_DRIVER_VERSION = '2.41'
+CHROME_DRIVER_VERSION = '77.0.3865.40'
```

Comment:
Is this intentional?

Reply:
Yes. This is done in order to fix  the issue mentioned [above](https://github.com/oppia/oppia/pull/9400#issuecomment-643424420).